tests/examples_methods_syntax/layered_chart_with_dual_axis.py

```diff
@@ -11,7 +11,7 @@
 source = data.seattle_weather()
 
 base = alt.Chart(source).encode(
-    alt.X('month(date):T').axis(title=None)
+    alt.X('month(date):T').title('')
```

Comment:
```suggestion
    alt.X('month(date):T').title(None)
```
Or there is another reason why you opt for `''`?